source/extensions/http/cache/file_system_http_cache/config.cc

```diff
@@ -56,9 +59,11 @@ class CacheSingleton : public Envoy::Singleton::Instance {
       cache = it->second.lock();
     }
     if (!cache) {
-      cache = std::make_shared<FileSystemHttpCache>(
-          singleton, std::move(config),
-          async_file_manager_factory_->getAsyncFileManager(config.manager_config()), stats_scope);
+      auto async_file_manager =
```

Comment:
nit: use an explicit type here (not obvious from context although I could guess).

Reply:
Done.